OpenOversight/tests/routes/route_helpers.py

```diff
@@ -75,31 +75,31 @@ def process_form_data(form_dict: dict) -> dict:
     """Mock the browser-flattening of a form containing embedded data."""
     new_dict = {}
     for key, value in form_dict.items():
-        if isinstance(value, list):
+        if type(value) is list:
```

Comment:
<img width="693" alt="Screenshot 2024-08-13 at 10 28 29 PM" src="https://github.com/user-attachments/assets/4efe2920-fa72-4099-a59f-ae4641863aef">

These is from: https://github.com/lucyparsons/OpenOversight/pull/1101/files